src/App.js

```diff
@@ -136,16 +96,16 @@ const App = () => {
           <img src={Search} alt="Search"/>
         </button>
       </div>
-      <SearchContainer
-        Annotations={Annotations}
-        annotManager={annotManager}
-        docViewer={docViewer}
-        searchTermRef={searchTerm}
-        searchContainerRef={searchContainerRef}
-        open={searchContainerOpen}
-      />
-      <div id="scroll-view" ref={scrollView}>
+      <div class="flexbox-container" id="scroll-view" ref={scrollView}>
```

Comment:
class -> className

Reply:
Haha good eye, already fixed in my new branch in https://github.com/PDFTron/webviewer-custom-ui/commit/7b8b8afde985d504160469040ab32d1652ee6bcc 👍